packages/@uppy/components/src/Dropzone.tsx

```diff
@@ -29,7 +29,6 @@ export default function Dropzone(props: DropzoneProps) {
       className="uppy-reset"
       data-uppy-element="dropzone"
       role="presentation"
-      tabIndex={0}
```

Comment:
I think I added this on purpose?

Reply:
Why was it added? I couldnt find the reason in blame. And it’s not a button so it shouldn’t have a tabIndex (which is what the linter says)